src/ocean/net/device/Berkeley.d

```diff
@@ -1353,7 +1355,14 @@ deprecated public class IPv4Address : Address
         static uint parse(istring addr)
         {
                 char[64] tmp;
-                return ntohl(inet_addr(toStringz(addr, tmp)));
+                if ((addr.length + 1) > tmp.length)
```

Comment:
Yeah... It shows that `ocean.text.util.StringC` doesn't really provide a convenient replacement for all usages of `ocean.stdc.stringz`.

This pattern is repeated 3 times in this module and I see similarly awkward code replacing a simple call in several other modules. I dunno. I don't really like, whether the modules in question could be deprecated or not. Presumably the same problem could occur in application code.